### PR TITLE
README corrections

### PR DESCRIPTION
**Pull request by colriot**
Fixed hopefully all broken links and formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ As you might have guessed, it's a DSL for Android. It is written in [Kotlin](htt
 	* [Why not Scaloid?](#why-not-scaloid)
 	* [Supporting existing code](#supporting-existing-code)
 	* [How it works](#how-it-works)
-	* [Is it extendable?](#is-it-extendable)
+	* [Is it extendable?](#is-it-extensible)
```
Comment (yanex) [inline README.md]: extendable → extensible

Comment (colriot) [inline README.md]: Woops. Missed that.

**Comment by yanex** [comment]
Thank you for the contribution! :penguin:

**Comment by colriot** [comment]
Welcome!

**Pull request merged**